guide/yaml/custom-entities.md

```diff
@@ -243,6 +243,14 @@ so that the `$message` we passed above gets logged and reported back:
           period: 1s
           command: tail -1 server-input
 
+##### Windows Command Sensor
+
+      brooklyn.initializers:
```

Comment:
I think there should be some text here describing what this is and what it does or a link to somewhere which describes it.